src/simpleweighteddigraph.jl

```diff
@@ -85,6 +93,10 @@ edgetype(::SimpleWeightedDiGraph{T, U}) where T<:Integer where U<:Real = SimpleW
 edges(g::SimpleWeightedDiGraph) = (SimpleWeightedEdge(x[2], x[1], x[3]) for x in zip(findnz(g.weights)...))
 weights(g::SimpleWeightedDiGraph) = g.weights'
 
+function outneighbors(g::SimpleWeightedDiGraph, v::Integer)
+    mat = g.weights
+    return view(mat.rowval, mat.colptr[v]:mat.colptr[v+1]-1)
+end
 inneighbors(g::SimpleWeightedDiGraph, v::Integer) = g.weights[v,:].nzind
```

Comment:
Can we use a `view` here?

Reply:
No, because a view has no field nzind. I think we can maybe come up with a faster implementation that allocate less, I will see that...

Reply:
but better to keep it like this for this PR I think

Reply:
https://github.com/JuliaGraphs/SimpleWeightedGraphs.jl/issues/31